sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.0.0-beta.4 (Unreleased)
 
+### Breaking Changes
+- Renamed all `TopCount` listing options to `MaximumPageSize`. Affected classes include `GetAlertsOptions`, `GetDataFeedsOptions`, `GetHooksOptions`, and others.
```

Comment:
I am all for renaming `top` to something more clear.
I was just looking at what other libraries use in .NET and saw that Search uses just `skip` and `size` for these properties, while Synapse uses `skip` and `limit`.
I am curious if the naming distinction was already discussed and approved, or this is a proposal.

Reply:
Tables named this `maxPerPage`.

https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/src/TableClient.cs#L781

Reply:
@kinelski looks like we might be all over the place. I will recommend starting an email thread with KC and Ted about naming convention for these 2 (or more) paging parameters and when possible (libraries that haven't GAd) be consistent.

Reply:
Updated name to `MaxPageSize` as suggested by KC.

Reply:
thank you for following up on this!